src/shared/views/Mapper/CardBody.js

```diff
@@ -588,6 +590,8 @@ class _CardBody extends React.PureComponent<Props, State> {
                         <TutorialBox
                             content={tutorialContent}
                             boxType={tutorialMode}
+                            bottomOffset="80%"
```

Comment:
again here you changed the placement of the tutorial box.

ahh, probably before you set it globally and now per project type?

Reply:
Yes, the value was hardcoded in the `TutorialBox` component, but I added a prop on the component so we can reuse it between different project types. The original placement is the same, I've just changed where the value comes from.